code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -295,14 +298,14 @@
 		if (is_special_character(user))
 			message_admins("[A] has been summoned in [ADMIN_VERBOSEJMP(user_turf)] by [ADMIN_LOOKUPFLW(user)], a non-antagonist")
 		A.area_type = user_area.type
-		A.telegraph_duration = 100
-		A.end_duration = 100
+		A.telegraph_duration = storm_telegraph_duration
+		A.end_duration = storm_duration
 
 	user.visible_message("<span class='warning'>[user] holds [src] skywards as red lightning crackles into the sky!</span>", \
 	"<span class='notice'>You hold [src] skyward, calling down a terrible storm!</span>")
 	playsound(user, 'sound/magic/staff_change.ogg', 200, FALSE)
 	A.telegraph()
-	storm_cooldown = world.time + 200
+	storm_cooldown = world.time + staff_cooldown
```

Comment:
`storm_cooldown` should probably be renamed `storm_nextuse` to better reflect what its used for